eq-author-api/utils/verifyPublisherRequest.js

```diff
@@ -0,0 +1,13 @@
+const yaml = require("js-yaml");
+const fs = require("fs");
+const jwt = require("jsonwebtoken");
+
+const keysFile = process.env.KEYS_FILE || "./keys.yml";
+
+module.exports = authHeader => {
+  const keysYaml = yaml.safeLoad(fs.readFileSync(keysFile, "utf8"));
+  const keysJson = JSON.parse(JSON.stringify(keysYaml));
+  const publisherVerificationKey = keysJson.keys.publisherVerificationKey.value;
```

Comment:
All this work to this point in this function could be moved out of this function and made static on the file. We probably shouldn't be running all the parsing for each request.